storey/sources.py

```diff
@@ -655,6 +655,7 @@ def _blocking_io_loop(self):
                                 self.context.logger.error(
                                     f"For {parsed_line} value of key {key_field} is None"
                                 )
+                self._dates_indexes = []
```

Comment:
This will override the dates_indexes from the __init__.
You don't want to do it if it wasn't initially empty.